sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -89,6 +89,9 @@ export function decompressResponsePolicy(): PipelinePolicy;
 // @public
 export const decompressResponsePolicyName = "decompressResponsePolicy";
 
+// @public
+export function defaultFailoverHostGenerator(readHosts?: string[], writeHosts?: string[]): FailoverHostGenerator;
```

Comment:
since we don't invoke this in the default policy, perhaps a descriptive name like `readWriteFailoverHostGenerator`?

Also I think if both args are optional we might consider doing this as an options object `{readHosts?: string[], writeHosts?: string[]}` to avoid the case of calling it with something like `(undefined, [...])` in certain cases

Reply:
Done!